### DISCUSSION TITLE
Query Related to technical onboarding

### DISCUSSION THREAD

**Discussion by SuchitraSahoo2003** [microsoft/SATechnicalOnboarding#68] [Questions]
I have edited the portoflio project and the url of my github repository is also opening properly but the url which I got from static web app of microsoft azure that's not working. when i click on that url it shows nothing. Kindly someone resolve the query.

here is the screenshot
![image](https://github.com/microsoft/SATechnicalOnboarding/assets/148522756/c94c9e93-b503-43a4-bdad-389cf384143f)

**Comment by AnthonyByansi** [comment]
Hi @SuchitraSahoo2003 

We kindly request that you provide clarification regarding the deployment of the static site. It appears there may be instances where your site is being deployed from a branch containing changes that have not yet been merged into the main branch.
